lib/tasks/data_import/custom_fields/custom_fields.rake

```diff
@@ -0,0 +1,139 @@
+namespace :import do
+  desc 'Import custom field from csv file to database. By default, looks for a file at lib/seeds/custom-fields.csv, or you can pass in a custom file path as an argument'
```

Comment:
oooh neat!!

Reply:
only really added that to make testing easier but it could come in handy! 😄